### ISSUE TITLE
Shift+tab doesn't work to go to previous input

### ISSUE THREAD

**Issue by justingolden21** [pstanoev/simple-svelte-autocomplete#123]
Shift+tab should go backwards, but it doesn't do anything once you get to the top.

I also think tab and shift+tab shouldn't navigate menu options, which should only be arrow keys, if you examine similar inputs across the internet and look at some UX

Thank you : )

**Comment by justingolden21** [comment]
I was also wondering (didn't want to keep opening tickets to avoid spamming your repo) if it's possible to get the input to `autofocus` on page load.

Also, what is the correct syntax for setting an `onchange` listener, such as `on:change={() => console.log('hi')}`

**Comment by justingolden21** [comment]
One more comment (sorry for the spam), but is there an option to select all text on click?

**Comment by justingolden21** [comment]
None of my custom styles are being applied as well

```css
<style lang="postcss">
	/* the main control */
	.autocomplete {
		background-color: green;
		border: 1px solid red;
	}
	/* the input list */
	.autocomplete-input {
		background-color: green;
		border: 1px solid red;
	}
	/* the dropdown list */
	.autocomplete-list {
		background-color: green;
		border: 1px solid red;
	}
	/* items in the dropdown list */
	.autocomplete-list-item {
		background-color: green;
		border: 1px solid red;
	}
</style>
```

**Comment by pstanoev** [comment]
Fixed in version 2.3.2. 
Tab and Shift + Tab no longer move through items and the dropdown is closed.
